katas/001_optimize_a_query.md

```diff
@@ -98,9 +98,7 @@ RRE uses it's own format of ratings, so in Quepid, click the _Export Case_ optio
 
 You then need to put this ratings into RRE.  First move any ratings file in `./rre/src/etc/ratings/` out of that directory, leaving them in `./rre` is fine.  Then take your freshly exported `Notebook_Computers_rre.json` file and put it in the ratings directory.  
 
-Now, lets look at the RRE setup.  Open up the `Notebook_Computers_rre.json` file and change the index property from `Notebook Computers` to `ecommerce` to match the name of our Solr index and save that change.
-
-We want to regression test our pre Querqy algorithm and our post Querqy algorithm.  So look at the template files in `./rre/src/etc/templates`.  As you can see v1.0 is just the simplistic query.   However v1.1 is using the Querqy enabled request handler.  This is going to let us measure the two against each other.
+Now, lets look at the RRE setup. We want to regression test our pre Querqy algorithm and our post Querqy algorithm.  So look at the template files in `./rre/src/etc/templates`.  As you can see v1.0 is just the simplistic query.   However v1.1 is using the Querqy enabled request handler.  This is going to let us measure the two against each other.
```

Comment:
Did you test RRE post all the changes?  Curious if it still works.   It's a bit "abandonware" so I wonder longer term if we keep it in the chorus stack.

Reply:
I did run RRE and it produced an output. Not sure if all results are equal to what the results looked like before but I recall at least improvements in some metrics.